partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,134 @@
+        with PartFactoryBrep.lock:
+            if (
+                PartFactoryBrep.count_inflight_simple < PartFactoryBrep.MIN_SIMPLE_INFLIGHT
+                or file_size < PartFactoryBrep.MIN_SUBPROCESS_FILE_SIZE
+            ):
+                PartFactoryBrep.count_inflight_simple += 1
+                return False
+            else:
+                PartFactoryBrep.count_inflight_subprocess += 1
+                return True
+
+    def _load_brep_directly(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_load_brep_directly`

(ANN202)